core/src/commonMain/kotlin/com/ustadmobile/core/domain/invite/ParseInviteUseCase.kt

```diff
@@ -0,0 +1,45 @@
+package com.ustadmobile.core.domain.invite
+
+import com.ustadmobile.core.domain.phonenumber.PhoneNumValidatorUseCase
+import com.ustadmobile.core.domain.validateemail.ValidateEmailUseCase
+import com.ustadmobile.core.viewmodel.clazz.inviteviaContact.InviteViaContactChip
+
+
+class ParseInviteUseCase {
+    var phoneNumValidatorUseCase: PhoneNumValidatorUseCase? = null
+    operator fun invoke(
+        text: String,
+        phoneValidatorUseCase: PhoneNumValidatorUseCase
+    ): List<InviteViaContactChip> {
+        phoneNumValidatorUseCase = phoneValidatorUseCase
+        val parts = text.split(",\\s*".toRegex())
```

Comment:
Better to use .split(",").map { it.trim() } . Regex not needed here - and it can be tricky to predict exactly what a regex will do in any given edge case.